src/index.ts

```diff
@@ -6,7 +6,7 @@ import { DocumentNode } from './dom/basicdom';
 declare global {
     export class SvelteComponent {
         $destroy(): void;
-        constructor(options: { target?: ViewNode | Element , props?: any, anchor?: ViewNode | Element, intro?: boolean });
+        constructor(options: typeof { target?: ViewNode | Element , props?: any, anchor?: ViewNode | Element, intro?: boolean });
```

Comment:
I am not sure how typeof is solving the problem here. The root cause is that the SvelteComponent type gets included from svelte's own types and has Element | ShadowRoot where ours only has ViewNode | Element so they clash. In my own code I just forcibly cast App to unknown then to our SvelteComponent type.

Reply:
Yes, it's the clash that actually gave me this trouble.
`typeof` will perform a kind of force casting on options, just like in `svelteNative` function's `startPage` argument.

Reply:
Thanks for this, I went with a shim in the types folder which will make *.svelte imports return the correct type in the first place.